webapp/src/Controller/API/JudgehostController.php

```diff
@@ -192,6 +192,7 @@ public function createJudgehostAction(Request $request) : array
         } else {
             $judgehost = new Judgehost();
             $judgehost->setHostname($hostname);
+            $judgehost->setActive((bool)$this->config->get('judgehost_enabled_by_default'));
```

Comment:
don't we need a `, true` here in the config get call?

Reply:
Nope, it uses `default`